### ISSUE TITLE
Media player progress indicator color contrast (WCAG 1.4.11) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#199]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:**
Course Overview page
Lesson 1: Overview of electronic Passport Next Generation
Lesson 2: Domestic Print
Lesson 3: Observation Print
Lesson 4: Manual QA
Lesson 5: Reports

**Description:**
The audio progress bar indicator in the section
   • "What is ePNG?"
has insufficient color contrast. The element was calculated to have the following traits (dark cyan/light grey - dark cyan/dark grey)
   • Foreground: `rgb(71, 136, 154)` - `rgb(50, 114, 131)`
   • Background: `rgb(178, 186, 192)` - `rgb(76, 76, 76)`
   • The contrast ratio is: 2:1 - 1.6:1

Any visual information necessary to indicate a state must also ensure that the information used to identify the control in that state has a minimum 3:1 contrast ratio.
*******************************************************************************
